src/components/ReportActionItem/ReportActionItemImage.tsx

```diff
@@ -50,22 +41,22 @@ const defaultProps = {
  * and optional preview modal as well.
  */
 
-function ReportActionItemImage({thumbnail, image, enablePreviewModal, transaction, canEditReceipt, isLocalFile}) {
+function ReportActionItemImage({thumbnail = null, image, enablePreviewModal = false, transaction, canEditReceipt = false, isLocalFile = false}: ReportActionItemImageProps) {
     const styles = useThemeStyles();
     const {translate} = useLocalize();
-    const imageSource = tryResolveUrlFromApiRoot(image || '');
-    const thumbnailSource = tryResolveUrlFromApiRoot(thumbnail || '');
-    const isEReceipt = !_.isEmpty(transaction) && TransactionUtils.hasEReceipt(transaction);
+    const imageSource = tryResolveUrlFromApiRoot(image ?? '');
+    const thumbnailSource = tryResolveUrlFromApiRoot(thumbnail ?? '');
+    const isEReceipt = transaction && TransactionUtils.hasEReceipt(transaction);
 
-    let receiptImageComponent;
+    let receiptImageComponent: ReactElement;
```

Comment:
Is this possible to use ReactNode here?
```suggestion
    let receiptImageComponent: ReactElement;
```

Reply:
But I thought ReactElement is more specific than ReactNode.

Reply:
I'm good with both, not a blocker 👍